refactored_code.rb

```diff
@@ -0,0 +1,43 @@
+###---Modules
+module Fivable
```

Comment:
throw each module into its on file. When reading 'refactored_code.rb' I have no idea what type of code I'll find in there. Plus - the parent super class in w the modules - it deserves its own file, too